gerritcheck/check.py

```diff
@@ -133,6 +133,12 @@ def cppcheck_on_files(files, commit):
             review["labels"] = {"Code-Review": -1}
             return json.dumps(review)
 
+    # Check the return code only just now as cppcheck might still have returned
+    # some valid comments.
+    if rc != 0:
```

Comment:
@grundprinzip It was added mainly to inform developer that change was not completely verified with cppcheck. Otherwise review receives "No issues found. Ok" now.

I was thinking about providing `verbose` mode, but with different purpose - to disable positive feedback (with positive feedback gerrit history becomes more dirty in our case).